### DISCUSSION TITLE
https config

### DISCUSSION THREAD

**Discussion by ionv06** [sqlpage/SQLPage#622] [Q&A]
Hi,
Is there any way to test a site with an ssl certificate on localhost? I generated the certificate with mkcert (https://mkcert.dev/), but I don't know how to configure SQLPage to use it (I added "https_domain":"localhost" to sqlpage.json and copied the two certificate files, but it doesn't work, I get an error when configuring https)
`ERROR sqlpage::webserver::https] Unable to configure HTTPS: Order(Acme(HttpRequest(Non2xxStatus { status_code: 400, body: "{\n  \"type\": \"urn:ietf:params:acme:error:rejectedIdentifier\",\n  \"detail\": \"Invalid identifiers requested :: Cannot issue for \\\"localhost\\\": Domain name needs at least one dot\",\n  \"status\": 400\n}" })))`
I also tried with domain with dot localhost.com but it still doesn't work

**Comment by lovasoa** [comment]
Hello, and sorry for the late reply.
  Currently, SQLPage does not support providing a custom SSL certificate for your web server. It only supports automatically generating a certificate from letsencrypt.
  If you have your own ssl certificate, you can [use a reverse proxy](https://sql.datapage.app/your-first-sql-website/nginx.sql) to terminate SSL connections in front of your SQLPage application.
